catalog/solutions/carbonblock/1.0.0/solution.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "carbonblock",
+  "version": "1.0.0",
+  "name": "CarbonBlock",
+  "website": "https://www.circulartree.com/product",
+  "provider": "CircularTree",
```

Comment:
Hello Anıl,
Thank you for submitting Circular Tree's solution. 
I would kindly ask you to replace the name of the provider so as to only include lowercase letters and dashes (e.g. `circular-tree` or `circulartree`).
Thank you!